BUILDING.md

```diff
@@ -20,7 +20,7 @@ If in doubt, use the most recent stable version of each build tool.
     `PERL_EXECUTABLE`.
     * To build without Perl (not recommended) see [this section.](#using-pre-generated-build-files)
 
-  * [Go](https://golang.org/dl/) 1.18 or later is required. If not found by
+  * [Go](https://golang.org/dl/) 1.17.13 or later is required. If not found by
```

Comment:
There is an opportunity here to specify that 1.17 works for building awslc but successfully running the tests requires Go 1.20. (https://github.com/aws/aws-lc/pull/1667#discussion_r1654965756) :)

Reply:
Addressed by fixing the test issue so everything works with Go 1.17